frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -27,6 +27,14 @@ type ReviewRow = {
 };
 
 const AdminDashboard = (): React.ReactElement => {
+  const [data, setData] = useState<ReviewResponse[]>([]);
+
+  useEffect(() => {
+    getReviews().then((allReviews: ReviewResponse[]) => {
```

Comment:
with the change of import, we would have to change this to `reviewApiClient.getReviews()`. The current use of .then() is completely valid. I prefer using await to make the code a bit more readable. If you want to await the response of getReviews, then you can look into this article: https://devtrium.com/posts/async-functions-useeffect

Reply:
Just wondering, is there other reasons for switching from then to await or would it just be for better readability?

Reply:
I only brought it up for consistency with the rest of the project. This should be perfectly valid too! :)